FetchMigration/python/migration_monitor.py

```diff
@@ -47,6 +49,9 @@ def check_if_complete(doc_count: Optional[int], in_flight: Optional[int], no_par
     # TODO Add a check for partitionsCompleted = indices
     if doc_count is not None and doc_count >= target:
         # Check for idle pipeline
+        logging.info("Target doc count reached, checking for idle pipeline...")
+        logging.debug("Idle pipeline metrics: " +
+                      ",".join(map(str, [in_flight, no_part_count, prev_no_part_count])))
```

Comment:
consider prefacing these values w/ labels.
Second question - what does it mean to be inFlight and what does "no_part" mean?  From looking through the existing code, my bet is that it's sortof like a reverse countdown - something like how many of the original partitions no longer need to be moved?  Consider a clearer name to self document this field.

Reply:
* `in_flight` is a [counter published by Data Prepper](https://github.com/opensearch-project/data-prepper/blob/main/docs/monitoring.md#plugin-metrics) to describe the number of records in memory that have been pulled from the source plugin and are currently being processed by the sink plugin
* The `no_part` variables track the number of times the Data Prepper source coordination implementation attempted to fetch a  partition to move data for and found none pending. This is our indicator that data migration has finished and the DP pipeline is idle. The DP maintainers advised us to the track the _rate of change_ of this value rather than just a non-zero value, hence the `prev` counter.

Incorporated PR feedback by adding labels and renaming `no_part` variables to `no_partition` for clarity.